src/e2eTest/kotlin/io/github/gradlenexus/publishplugin/e2e/NexusPublishE2ETests.kt

```diff
@@ -47,4 +47,40 @@ class NexusPublishE2ETests : BaseGradleTest() {
             assertSuccess(":releaseSonatypeStagingRepository")
         }
     }
+
+    @ParameterizedTest(name = "{0}")
+    @ValueSource(strings = ["nexus-publish-e2e-minimal", "nexus-publish-e2e-multi-project"])
+    fun `release project to real Sonatype Nexus in two executions`(projectName: String) {
```

Comment:
1. It will most likely fails, as the version is [timestamp based](https://github.com/gradle-nexus/nexus-publish-e2e-minimal/blob/6a7fc8d7895b0f3aea803bf7037cef05bfa1d438/build.gradle#L9-L11) to avoid problem described in https://github.com/gradle-nexus/publish-plugin/pull/153 (in addition the version is not properly set in the multi-module e2e project as I mentioned in the comment a few days ago).
2. I can run it from a side branch to verify (if you think `1.` is wrong or once there is a workaround - what do you propose?), but in general, I'm still not sure, if we should call it on every merge/push to master, as e2e tests (despite their name) are rather smoke/sanity tests for critical scenarios.

Reply:
>as the version is timestamp based to avoid problem described

What do you think of passing the version on the command line instead?

>I'm still not sure, if we should call it on every merge/push to master, as e2e tests (despite their name) are rather smoke/sanity tests for critical scenarios.

There are only 2 merges to master per year. Is adding a test really a problem?
If you absolutely want to reduce the number of interactions with Central, you might consider launching a random subset of tests every time.

Reply:
> There are only 2 merges to master per year. Is adding a test really a problem?

It depends of a time window. I see 17 commits to master in 2 weeks :). With 2x30+ projects releases released to Nexus.

In general, this is not a big problem right now (especially that I plan to reduce number of e2e tests execution #181 to mainly master and cron), but I'm really happy being able to perform the real e2e testing with the real Sonatype Nexus and it would be problematic to get banned (or something). E2E test should cover the main/critical scenarios and from my experience, especially in the "enterprise stuff", they tend to grow (with all the maintenance issues). I'm really curious, how many project will use the new mode (and publishing performed in two phases).

Anyway, let's see how many build it trigger in the coming weeks. In the worst case, we could add a switch to just call them (the new one) from cron, not for every master build.

Reply:
> What do you think of passing the version on the command line instead?

Yes, give me a moment, I'm playing with some idea.

Reply:
Would adding `-PoverriddenVersion=somethingUnique` twice with the same (unique) value generated in `given` be enough?

https://github.com/gradle-nexus/nexus-publish-e2e-minimal/commit/6a6fbdb3b34a8cacc54b634cf75031050f738a74
https://github.com/gradle-nexus/nexus-publish-e2e-multi-project/commit/a2de98f251529bee6e567f742828febd34f99db9

Reply:
Btw, how would you approach the issue with:

> :nexus-publish-e2e-multi-project:unspecified

reported for the root project? Here, in the multi-module Kotlin project, the things are slightly complicated as the version is set in `buildSrc/library-publishing-conventions.gradle.kts` just for submodules?